### ISSUE TITLE
Endgame Power: Black Hole Power

### ISSUE THREAD

**Issue by General-Ike** [GTNewHorizons/GT-New-Horizons-Modpack#7722]
Disclaimer: I'm LuV, pre-fusion.

TL;DR: Special space station power generator.

I've heard fusion power gets pretty stale in the later game. My suggestion is for one of the zpm+ tiers of power gen to be black hole power (if you're unfamiliar, you can look it up, but it's basically capturing accelerated plasma escaping from the ergosphere of a black hole).

The basic concept is that either a special rocket (or regular high tier rocket), allows you to travel to a black hole "system" on the galacticraft map. There, you can choose to set up a space station with a special cost (gravitation engine, alcubierre drive, computer cubes, heavy reinforced plates, containment casings, etc). From that space station, you can collect accelerated plasma which can be sent back to your base for power gen.

The complexity should be in operating a space station near the event horizon of a black hole (not the actual collection). Since I think there's good potential with this idea, I'll not limit anyone's creativity by getting to specific, but I could imagine operation might require a fuel/energy supply, an OC control program, and "special" maintenance operations.

Additional ideas:
-Space ship is destroyed upon arrival
-Cant launch spaceships from station, requires teleportation
-Add some risk. Make a time limit on getting your station "running", or ongoing maintenance issues cause your station to fall into the black hole

Yes, space station code is super buggy/laggy, this would need to be addressed (which would make space stations viable as a bonus. Or you could treat some new dimension like a space station, or some other workaround). Or maybe it makes this idea simply impossible or impractical from a coding perspective.

#
#### What is your GTNH Discord username?
Commander Crankshaft

**Comment by Nicouuuuu** [comment]
I like this idea, 
I thought about something similar but with neutron stars. Like you can gather the beam from one of the pole.
Then it could be used to "craft" like a very tiny and stable star in the player base, which will output a huge amount of power.
Of course this is something that should be gated very late game like the idea of General-Ike, or even later

**Comment by johnch18** [comment]
I'd prefer we add more plasmas first. Right now it seems that you use either Helium or Tin plasma and nothing else, and the automation for it is pretty boring.

**Comment by General-Ike** [comment]
I'd aimed for a power source that is unique both to set up and operate (and preferably unique in concept). Something to excite/engage a player when they think about progression. Like the very first time you build an EBF, launch a rocket, or create a part in the assy-line. Why shouldn't endgame power be as, or more, engaging than previous advancements.

I think new plasma might be good, it doesn't really tread on new ground.

**Comment by Prometheus0000** [comment]
You can talk to 0lafe about plasma. I think he had 2/3 tickets on it? They were stale though, so they got closed. Not that it's a bad idea or anything. Also, adding seems like it might not be the right idea, changing existing ones might be better. Or not, whatever.

**Comment by yukieiji** [comment]
There were a few threads on end game power generation, but they were tagged as stale and closed.
Some of the discussions included plasma, black hole generators, and multi naquada reactors.
#3551 #6294 #6545
